docs/static/ls-ls-config.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[ls-to-ls]]
+=== Configuring Logstash-to-Logstash Communication
+
+In a complex network environment, you may need two Logstash machines talking to each other. That is, the output from the first Logstash machine becomes the input for the second Logstash machine. You may need this configuration if the data path crosses network or firewall boundaries, for example.
+
+NOTE: Don't complicate your configuration unnecessarily. If you don't have a compelling need for Logstash-to-Logstash communication, then don't implement it.
```

Comment:
Do we have a way to format this as an aside? Otherwise, I'd just gently incorporate it into the preceding paragraph.